core/screenshot-testing/build.gradle.kts

```diff
@@ -24,11 +24,10 @@ android {
 }
 
 dependencies {
+    api(libs.androidx.activity.compose)
```

Comment:
Why `api` and not `implementation`? Is a module which is dependent on `:core:screenshot-testing` using this library's API?

Same question for the other 2 `api` dependences.